src/Exception/UnsupportedValueException.php

```diff
@@ -24,29 +24,25 @@
 
 class UnsupportedValueException extends InvalidArgumentException implements Exception
 {
-    /** @var mixed */
-    private $value;
+    private mixed $value;
```

Comment:
You should add CPP for this one as you are already changing the property definition and the contructor.

Reply:
I enabled the CPP rule and fixed the two mistakes that the automatic fixer made. CPP is now used wherever possible.